22_pipeslab/pipeslab.txt

```diff
@@ -3,19 +3,42 @@ For each of the following, type a single command line that performs:
   1. Print the number of lines from "ls -l" which represent a path that has execute permission to their owner.
   Answer: 
 
+  tocode@tocode-VirtualBox:~$ ls -alF | grep tocode | cut -c4 | grep x | wc | cut -d' ' -f6
```

Comment:
hmm.. 

```grep tocode``` is not needed here because we want "execute permission to **their** owner". not just for your user... 

```wc | cut -d' ' -f6``` using cut here is not needed because you can just ask ```wc``` to 
print the lines with ```wc -l```  also I kinda hate the -f option with cut because it's too flaky.. 
for example on my machine it works with ```-f7```. so I don't recommend  using cut  to often.